packages/directory/addon/components/dir-table.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright 2018, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ *
+ * Usage:
+ * {{dir-table
+ *   items=items
+ * }}
+ */
+import Component from '@ember/component';
+import { computed, get, getWithDefault } from '@ember/object';
+import layout from '../templates/components/dir-table';
+import Table from 'ember-light-table';
+import Moment from 'moment';
+
+export default Component.extend({
+  layout,
+
+  /**
+   * @property {Array} classNames
+   */
+  classNames: ['dir-table'],
+
+  /**
+   * @property {String} height - height of the table
+   */
+  height: '45vh',
```

Comment:
What is this height used for? Couldn't find this in the light-table documentation.